hazelcast/src/test/java/com/hazelcast/util/FilteringClassLoader.java

```diff
@@ -60,28 +62,25 @@ public FilteringClassLoader(List<String> excludePackages, String enforcedSelfLoa
 
     @Override
     public URL getResource(String name) {
-        return delegatingClassLoader.getResource(name);
+        return checkResourceExcluded(name) ? null : delegatingClassLoader.getResource(name);
```

Comment:
AFAIK the appropriate name for the delegatingClassLoader should be delegateClassLoader. The delegating is the task of the FilteringClassLoader; the wrapped classloader is the delegate. But this can be fixed in a different PR.